device/virtual/x86_64-kvm_x86_64-r0/virtual_chassis/common/port_config.ini

```diff
@@ -0,0 +1,33 @@
+# name          lanes             alias             index       speed
```

Comment:
Can we avoid defining these files and instead use the prebuilt ones inside the vs docker image?

Reply:
ack